Rakefile

```diff
@@ -103,7 +103,7 @@ namespace :test do
           "bundle exec appraisal #{ruby_runtime}-#{appraisal_group} rake #{spec_task}"
```

Comment:
changes in this file are unrelated, just enjoying our own Datadog's static code analysis tool